tests/integration/defs/perf/agentx_client.py

```
@@ -0,0 +1,538 @@
+def _run(args: argparse.Namespace, artifact_dir: str) -> int:
+    url = f"http://{args.host}:{args.port}"
+    # Custom tokenizers (e.g. deepseek_v4) are loaded in aiperf's worker
+    # processes, which do not inherit a CLI trust flag.
+    os.environ["HF_HUB_TRUST_REMOTE_CODE"] = "1"
+
+    ensure_aiperf(url, args.model)
+    verify_endpoint(url, args.model)
+
+    cmd = build_aiperf_cmd(args, url, artifact_dir)
+    _log("running: " + " ".join(cmd))
+    completed = subprocess.run(cmd)
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)